README.md

```diff
@@ -1,4 +1,9 @@
-![gofannon logo](./gofannon.jpg)  
+![gofannon logo](https://github.com/The-AI-Alliance/gofannon/blob/main/gofannon.jpg)  
+![CI](https://github.com/The-AI-Alliance/gofannon/actions/workflows/main.yml/badge.svg)
```

Comment:
Good find- but this workflow doesn't exist so this badge won't work- I'm not sure what workflow we have that would work for it.

Reply:
You know what- comment this out @jolson-ibm when we get the integration tests going, we'll plug them in here- this is a badge that says if they're passing or not.